aws/resource_aws_guardduty_member_test.go

```diff
@@ -33,6 +35,47 @@ func testAccAwsGuardDutyMember_basic(t *testing.T) {
 	})
 }
 
+func testAccAwsGuardDutyMember_invite(t *testing.T) {
+	resourceName := "aws_guardduty_member.test"
+	accountID := "111111111111"
```

Comment:
In order to acceptance test these properly, I will move them to environment variables, e.g.

```
accountID := os.Getenv("AWS_GUARDDUTY_MEMBER_ACCOUNT_ID")
if accountID == "" {
  t.Skip("... helpful message here ...")
}
email := os.Getenv("AWS_GUARDDUTY_MEMBER_EMAIL")
if email == "" {
  t.Skip("... helpful message here ...")
}
```